1682/GasPoweredHALE.py

```diff
@@ -0,0 +1,31 @@
+from gas_hale import GasPoweredHALE
```

Comment:
great! Maybe move it into `gas_hale.py` and stick it in a method like
```python
def make_sensitivity_plots():
    # code goes here
```
You can then call `make_sensitivity_plots()` from the `if __name__ == "__main__"` section of gas_hale.py, or import `make_sensitivity_plots()` into an ipython notebook and use it there